README.md

```diff
@@ -105,8 +105,8 @@ optional arguments:
   --count COUNT_LABEL   Count and display the number of specified objects on the frame. You can enter either the name of the object or its label id (number).
   -dev DEVICE_ID, --device_id DEVICE_ID
                         DepthAI MX id of the device to connect to. Use the word 'list' to show all devices and exit.
-  -lowb, --low_bandwidth
-                        Enable low bandwidth mode that uses encoding for data transfer to limit it's size and increase throughput
+  -bandw {auto,low,high}, --bandwidth {auto,low,high}
+                        Force bandwidth mode. Default: auto. Choices: auto, low, high
```

Comment:
Few more words to explain the options could help, e.g `low` uses MJPEG encoding, `high` streams uncompressed, `auto` defaults to low bandwidth for USB2 or network/PoE.
And could remove `Choices: ..`, as already printed on the first line.